README.md

```diff
@@ -150,7 +150,7 @@ Each day I take one subject from the list below, read it cover to cover, take no
 - [x] [The Machine Learning Mastery Method](http://machinelearningmastery.com/machine-learning-mastery-method/)
 - [x] [Machine Learning for Programmers](http://machinelearningmastery.com/machine-learning-for-programmers/)
 - [x] [Applied Machine Learning with Machine Learning Mastery](http://machinelearningmastery.com/start-here/)
-- [ ] [Python Machine Learning Mini-Course](http://machinelearningmastery.com/python-machine-learning-mini-course/)
+- [ ] [(In Progress) Python Machine Learning Mini-Course](http://machinelearningmastery.com/python-machine-learning-mini-course/)
```

Comment:
Found a good article about standardizing and normalizing data:
http://www.dataminingblog.com/standardization-vs-normalization/

Normalizing is converting the range of a feature to [0,1]. The simple formula:
x_new = (x - x_min) / (x_max - x_min)
This will always result in a fraction between 0 and 1, where the denominator is the original scale and the numerator is the pre-scaled value.

Standardizing is the process of making zero the 'mean' and applying unit variance.
x_new = x - mean(x) / std(x) 

Found this neat visualization of standard deviation (Actually, image depicts variance, but stdev is square root of variance) https://en.wikipedia.org/wiki/Standard_deviation#/media/File:Variance_visualisation.svg